### PR TITLE
Migrating from contrib.slim to tf_slim

### PR DESCRIPTION
**Pull request by marksandler2**
# Description
Bringing in internal changes. Moves slim from contrib.slim to tf_slim, updates some internal implementation details in MobilenetV3. 

## Type of change

Upgrading research.slim to eliminate most dependencies on contrib.slim - a few are still lingering particularly in some dusty corners of quantization and training. 

Note: Please delete options that are not relevant.

- [x] Documentation update
- [x] TensorFlow 2 migration
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [x] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [x] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [x] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: research/slim/nets/mobilenet/README.md [hunk 1]
```diff
@@ -103,36 +103,88 @@ tool.
...
 
 ## Training
 
 ### V3
+The following configuration, achieves 74.6% using 8 GPU setup and 75.2% using
+2x2 TPU setup.
+
+                          |    8 GPU setup  | Notes  |
+-------------------------- | ------------------| ----------|----|
+Final Top 1 Accuracy       |      74.6   |    |
```
Comment (jaeyounkim) [inline research/slim/nets/mobilenet/README.md:167]: Please fix this table.

File: research/slim/nets/mobilenet/README.md [hunk 2]
```diff
@@ -103,36 +103,88 @@ tool.
...
+[float_v2_1.3_224](https://storage.googleapis.com/mobilenet_v2/checkpoints/mobilenet_v2_1.3_224.tgz) |[uint8][quantized_v2_1.3_224]   | 509      | 5.34           | 74.4           | 92.1           | 123.0
+[float_v2_1.0_224](https://storage.googleapis.com/mobilenet_v2/checkpoints/mobilenet_v2_1.0_224.tgz)  |[uint8] [quantized_v2_1.0_224]  | 300      | 3.47           | 71.8           | 91.0           | 73.8
+[float_v2_1.0_192](https://storage.googleapis.com/mobilenet_v2/checkpoints/mobilenet_v2_1.0_192.tgz)  |[uint8] [quantized_v2_1.0_192]  | 221      | 3.47           | 70.7           | 90.1           | 55.1
+[float_v2_1.0_160](https://storage.googleapis.com/mobilenet_v2/checkpoints/mobilenet_v2_1.0_160.tgz) | [uint8] [quantized_v2_1.0_160]  | 154      | 3.47           | 68.8           | 89.0           | 40.2
+[float_v2_1.0_128](https://storage.googleapis.com/mobilenet_v2/checkpoints/mobilenet_v2_1.0_128.tgz) |  [uint8][quantized_v2_1.0_128]  | 99       | 3.47           | 65.3           | 86.9           | 27.6
```
Comment (jaeyounkim) [inline research/slim/nets/mobilenet/README.md:113]: Can you update the "Quantized" column? Some rows show [uint8] quantized_v2_x.xx_xxx or only uint8.

File: research/slim/README.md [hunk 3]
```diff
@@ -18,15 +15,15 @@ data reading and queueing utilities. You can easily train any model on any of
...
+For developing or modifying your own models, see also the [main TF-Slim page](https://github.com/google-research/tf-slim/tree/master/tf_slim).
 
 ## Contacts
 
 Maintainers of TF-slim:
 
-* Sergio Guadarrama, GitHub: [sguada](https://github.com/sguada)
 * Nathan Silberman,
-  GitHub: [nathansilberman](https://github.com/nathansilberman)
+  github: [nathansilberman](https://github.com/nathansilberman)
```
Comment (jaeyounkim) [inline research/slim/README.md:25]: Can you remove Nathan Silberman if he is not maintaining this repository?

File: research/slim/nets/mobilenet/README.md (hunk 1 above)
Comment (marksandler2) [reply research/slim/nets/mobilenet/README.md]: Done

**Pull request closed without merging**